spec/oga/xml/document_spec.rb

```diff
@@ -42,6 +42,27 @@
     end
   end
 
+  describe '#root_element' do
+    before do
+      @document = described_class.new
+    end
+
+    it 'returns first element' do
+      child = Oga::XML::Element.new(:name => 'foo')
+      @document.children << child
+      @document.root_element.should == child
+      @document.root_element.name.should == 'foo'
+    end
+
+    it 'returns first element even when other nodes present' do
```

Comment:
Please use `it 'returns the first element even when it is preceded by another node'`